src/server/internalServer.cpp

```diff
@@ -1184,6 +1207,13 @@ std::unique_ptr<Response> InternalServer::handle_content(const RequestContext& r
     auto response = ItemResponse::build(*this, request, entry.getItem());
     response->set_etag_body(archiveUuid);
 
+    if ( !startsWith(entry.getItem().getMimetype(), "application/pdf") ) {
```

Comment:
Are we confident that all PDFs will have a set MIME type, and that it will always be `application/pdf` (and always lowercase)? What about `text/pdf` or `application/x-pdf`, which can be used? I'm thinking of Zimit archives, where we don't really control such things (for Gutenberg and mwOffliner, we can be sure of the MIME type, I think).

Reply:
I think @mgautierfr and/or @rgaudin can answer this question better than me

Reply:
That's convincing to me!